Automatic Validation ended with:
> 2024-01-18T20:02:34.3174763Z ##[error] One or more errors occurred. (Manifest is invalid.) Manifest is invalid. (Line: 11, Col: 1, Idx: 308) - (Line: 11, Col: 2, Idx: 309): While parsing a block mapping, did not find expected key.

(Automated response - build 615.)

--- a/manifests/d/DarkfullDante/srtAlign/0.6.1/DarkfullDante.srtAlign.installer.yaml
+++ b/manifests/d/DarkfullDante/srtAlign/0.6.1/DarkfullDante.srtAlign.installer.yaml
@@ -4,10 +4,10 @@
 
 PackageIdentifier: DarkfullDante.srtAlign
 PackageVersion: 0.6.1
-Installers:
 Dependencies:
   PackageDependencies:
   - PackageIdentifier: Microsoft.DotNet.Runtime.3_1
+Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/fcusson/srt-align/releases/download/v0.6.1/srt-align.exe
   InstallerSha256: 9125360df1d6ad77b33bfd92fd23e6e1f62dea13d3e7b5dddd2cd3b25fc94945

